### DISCUSSION TITLE
Experimental parser still experimental?

### DISCUSSION THREAD

**Discussion by thomasaarholt** [dbt-labs/dbt-core#9535] [Q&A] [answered]
Is the `dbt run -s mymodel --use-experimental-parser` flag still experimental? How does it perform compared to the default parser?

**Comment by jtcohen6** [comment, marked as answer]
Hey @thomasaarholt! There isn't (shouldn't be) any difference. We haven't made any "experimental" updates since 2021 (in the leadup to dbt-core v1.0). But we've kept the flag around for the _next_ time we want to try some more-radical changes, and need a way to:
- (a) let folks opt in/out
- (b) compare timing/outputs

**Reply by thomasaarholt** [reply]
Thanks! In that case, you may want to update this documentation: https://docs.getdbt.com/reference/parsing
> An experimental parser, which extracts information from simple models much more quickly

**Reply by dbeatty10** [reply]
Good call @thomasaarholt !

Opened this update to our documentation as a result: https://github.com/dbt-labs/docs.getdbt.com/pull/5147/
